source/ibm-colors-v1.8.sketchpalette

```diff
@@ -0,0 +1 @@
+﻿{"pluginVersion":"1.0","colors":["#ffffff","#000000","#cbebff","#8dd2ff","#6bbafb","#66a9eb","#518dca","#407093","#315d73","#26465b","#1a3645","#010204","#c1f2f5","#8ce6ef","#59cddb","#35b2c0","#20929e","#187c86","#08636b","#09474d","#033339","#010202","#b4f9eb","#7defe0","#4adcce","#00bfb0","#009584","#007f70","#00665a","#004c41","#00382d","#000202","#c8f08f","#b4e051","#8cd211","#5aa700","#4b8400","#2d660a","#144d14","#0a3c02","#0c2808","#010200","#feeb88","#fedb00","#f3ca00","#caa900","#9e8500","#867100","#6a5b00","#4d4000","#352800","#020100","#ffdcaf","#ffb586","#ff8d40","#ff6800","#e15702","#b64b31","#9a3b11","#821d11","#56170b","#020100","#ffdce4","#ffb6c1","#ff9399","#ff6962","#ee3640","#bd2931","#9f1f25","#831528","#60111e","#030001","#ffddff","#ffb3f1","#ff8cdc","#ff5bb0","#e44392","#b73d81","#902f6b","#751f58","#4b123c","#030102","#f2dcff","#e0bcff","#c7a5f9","#be88ed","#aa70dc","#8757a9","#6a4385","#533369","#412653","#020102","#e4e5ff","#c1c4ff","#abafff","#9499f3","#7c81dc","#6063ab","#4d5086","#383a6d","#202041","#010102","#e6e6e6","#d1d1d1","#bcbcbc","#a6a6a6","#8a898a","#6d6d6d","#585858","#414141","#171717","#000000","#e9e0e0","#d0c7c7","#b8aeae","#9e9494","#7d7373","#645a5a","#504646","#3c3232","#1a1314","#030000","#dfe9e9","#c8d2d2","#aeb8b8","#959f9f","#6d7777","#5a6464","#3c4646","#323c3c","#0d1111","#000203","#fdfdfd","#f9f9f9","#f4f4f4","#ececec","#fbfcfc","#f9f9fb","#f0f2f4","#ecf0f2","#fbfbfb","#fdfbfb","#f7f5f5","#f2eeee"]}
```

Comment:
Why did you include the version number in the file name? I think it would be better if we let that be in the package.json. One more things to touch and have to ensure always match.